book/creating-a-basic-website/adding-a-theme.rst

```diff
@@ -95,5 +100,18 @@ The next table shows the standard commands available with its parameters.
 |         | h: the height of the new image    |
 +---------+-----------------------------------+
 
+Global Image Compression
+-----------------------
```

Comment:
The underline should be as long as the title (then the test will also pass :wink:)